backend/src/test_example/entities/animals.entity.ts

```diff
@@ -6,5 +7,6 @@ export class AnimalEntity {
 	id: number;
 
 	@Column({ unique: true })
```

Comment:
`@IsNotEmpy `can be removed here, then  think we can merge!

Reply:
Already removed nice! ;D

Reply:
done